### ISSUE TITLE
Morrowind: tiny black window > black screen > close both

### ISSUE THREAD

**Issue by ArtMidir** [NVIDIAGameWorks/rtx-remix#72]
CPU: AMD Ryzen 5500
GPU: RTX 2080 Ti

GAME: The Elder Scrolls 3: Morrowind
Version: Steam + Mods

RTX Remix: 0.1.1

ISSUE: Can't launch game. When open all 3 log files successfully created (Attached to message) then appears RTX Remix console and tiny black window titled "Morrowind". After 1 second screen makes black [game tries to launch?] holds about 2 seconds, then all windows closes and i see Explorer, where i begin my adventure.

[Attached files on my OneDrive](https://1drv.ms/f/s!AvkEocg_aKtjhMBE28zrm-4ZJYUOvA?e=BQty5jl)

**Comment by svelle** [comment]
@ArtMidir have you disabled shaders in the launcher settings?

**Comment by ArtMidir** [comment]
@svelle unfortunately that doesn't help either...
Maybe there are other settings that can start it on?

**Comment by ArtMidir** [comment]
Just changed the settings and now managed to launch the game.
**However, the overlay does not appear.**

Now it's same situation as it was with Warcraft 3. The game works - there is no overlay.

**Comment by gkalian** [comment]
Make the game windowed. 
Did you also install wrapper dlls? It's not enough just to install rtx remix, you also need additional dlls and some rtx config. I found them in discord.

**Comment by svelle** [comment]
Right, try with the d3d8to9 linked in the user guide. Just drop the dll in the MW directory next to rtxr

**Comment by gkalian** [comment]
https://gist.github.com/gkalian/da5926253b85bc8d9357feb2df0f701d

I just made a small gist with kinda instructions i've gathered from discord. Might be useful probably.

**Comment by ArtMidir** [comment]
@svelle @gkalian Guys, thanks for the instructions. I did everything you said including:
https://gist.github.com/gkalian/da5926253b85bc8d9357feb2df0f701d
but the overlay doesn't show up.
Maybe I need to install https://github.com/NVIDIAGameWorks/dxvk-remix/ or is it completely different?

**Comment by gkalian** [comment]
Nope, it's non-compiled tool. Is your Morrowind version mods-free? Because it should work only on vanilla game. 
If you installed everything correctly, you should be able to see  RTX Remix info message above. And then alt+x should show you overlay. If it's not working, then I have no clue at the moment.

**Comment by ArtMidir** [comment]
@gkalian You know, I think there is a factor associated with software and hardware. The tool is only beta now.
Yes, I have mods, at least a fixed 16:9 screen ratio.
Hope with next patch it gonna work, because i want to try Warcraft 3 with RTX so much 😀

**Comment by gkalian** [comment]
If you have MGE XE installed, then Remix wont work. You better try on vanilla MW.
But you right, it's beta now,

**Comment by NV-LL** [comment]
Hello! Will you please test this on the latest release and let us know if the issue persists? Thank you!
